orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/QueueShovel.kt

```diff
@@ -34,15 +38,34 @@ class QueueShovel(
   private val queue: Queue,
   private val previousQueue: Queue,
   private val registry: Registry,
-  private val activator: Activator
+  private val activator: Activator,
+  private val config: DynamicConfigService
 ) {
   private val log = LoggerFactory.getLogger(javaClass)
 
   private val pollOpsRateId = registry.createId("orca.nu.shovel.pollOpsRate")
   private val shoveledMessageId = registry.createId("orca.nu.shovel.pushedMessageRate")
   private val shovelErrorId = registry.createId("orca.nu.shovel.pushedMessageErrorRate")
 
-  @Scheduled(fixedDelayString = "\${queue.shovel.poll-frequency.ms:50}")
+  @Scheduled(fixedDelay = 5_000)
+  fun pollStatus() {
+    if (isActive()) {
+      thread(name = "queue-shovel") {
```

Comment:
why does this go on a separate thread? what if that thread takes longer than 5s, won't spring just kick off the next one? is that ok?
seems that `migrateSome` can just go inside this method and it would be simpler

Reply:
yeah I'll rename, that's because at the time my mind was really set on how to check if we were active or not

Reply:
looks better in 5c36aa71d9c9b5b1328bf69f06487c587436420e?